src/PhpWord/Style/TextBox.php

```diff
@@ -66,6 +66,33 @@ class TextBox extends Image
      */
     private $borderColor;
 
+    /**
+     * background color.
+     *
+     * @var string
+     */
+    private $bgColor;
+
+    /**
+     * Set background color.
+     *
+     * @param string $value
+     */
+    public function setBgColor($value = null): void
```

Comment:
```suggestion
    public function setBgColor(string $value = null): self
```

Reply:
Return $this in a setter method is not used in any other styling class' setter method. Doing this for only one class or even worser for only one class method is very inconsistent. I think this should be changed for all setters in version 1.1.x.